public/javascripts/SVLabel/src/SVLabel/label/LabelContainer.js

```diff
@@ -64,6 +65,89 @@ function LabelContainer($) {
             });
     };
 
+    this.fetchLabelsToResumeMission = function (regionId, callback) {
+        $.getJSON(
+            '/label/resumeMission',
+            { regionId: regionId },
+            function (result) {
+                let labelArr = result.labels;
+                let len = labelArr.length;
+                for (let i = 0; i < len; i++) {
+                    console.log(labelArr[i].labelId);
+                    let povChange = svl.map.getPovChangeStatus();
+
+                    // Temporarily change pov change status to true so that
+                    // we can use util function to calculate the canvas coordinate
+                    // to place label upon rerender
+                    povChange["status"] = true;
```

Comment:
Basically, without this status change, when we render the labels on the screen, it does so assuming the starting orientation is the orientation at which the user had placed the label prior (which is not guaranteed). In order to avoid the weird scenario where the labels are placed incorrectly when first rendered then quickly snap back to place after the user adjusts the orientation (triggering the povChange status), we simulate this movement on load so the labels start in the correct position. 

Maybe there's a better way to do this?

Some screenshots to show the diff:
With the line in question:
![withpovchangeline](https://user-images.githubusercontent.com/37346427/107999752-12269880-6f9d-11eb-8e06-60e365cfeeed.PNG)

Without the line in question (notice how the labels are just floating - this is where the labels would be if the start orientation was the same as the orientation the user was at when the label was placed, even though it isn't):
![withoutpovchangeline](https://user-images.githubusercontent.com/37346427/107999806-36827500-6f9d-11eb-9fd9-f43c7cabaed3.PNG)

Reply:
Ohhh okay that makes sense! Could you update the comment to make that a bit more clear? But I think the implementation is fine :ok_hand:

Reply:
Done